src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,173 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    static class Node<T> {
+
+        private T value;
+        private Node<T> prev;
+        private Node<T> next;
+
+        Node(T data) {
+            this.value = data;
+            this.prev = null;
+            this.next = null;
+        }
+    }
```

Comment:
Don't use class Objects. Consider removing its import since it's not recommended to use it for the `remove(T object)` method.

Reply:
ok, done